pkg/auth/authenticator/password/ldappassword/url.go

```diff
@@ -0,0 +1,141 @@
+package ldappassword
+
+import (
+	"fmt"
+	"net"
+	"net/url"
+	"strings"
+
+	"github.com/go-ldap/ldap"
+)
+
+// LDAPURL holds a parsed RFC 2255 URL
+type LDAPURL struct {
+	// Scheme is ldap or ldaps
+	Scheme string
+	// HostPort is the host:port of the LDAP server
+	HostPort string
```

Comment:
I was sort of mirroring URL... they get parsed together in `url.Parse`, they get used together in `Dial`...